docs/geedocs/answer/4492617.html

```diff
@@ -13,55 +13,24 @@
 </div>
 <div class="content"> <a name="top_of_file"></a>
   <p><img src="../art/common/googlelogo_color_260x88dp.png" width="130" height="44" alt="Google logo" /></p>
   <h1><a href="../index.html">Google Earth Enterprise Documentation Home</a> | Install Google Earth Enterprise</h1>
   <h2>Uninstalling</h2>
-  <ul class="toc">
-    <li><a href="#overview">Overview</a></li>
-    <li><a href="#Uninstalling">Uninstalling</a></li>
-  </ul>
-  <hr>
-  <h2 id="overview">Overview</h2>
-  <p>This section provides instructions for uninstalling all or some of the
-    software provided in the Google Earth Enterprise. For
-    this procedure, you use the same installer you used to install the software. </p>
-  <div class="note"><strong>Note:</strong> As with installation, you can use either
-    the command-line uninstaller or the graphical user interface (GUI) uninstaller
-    to remove the Google Earth Enterprise Fusion and Server products.</div>
-  <hr>
-  <h2>Uninstalling</h2>
-  <p>To uninstall all or  part of the Google Earth Enterprise software:</p>
-  <ol class="spaced-list">
-    <li>Open a terminal window on your Linux workstation. </li>
-    <li>Log in as <code>root</code>.</li>
-    <li>Stop the Google Earth Enterprise Fusion system manager and Google Earth
-      Enterprise Server, if you have a previous version installed and they
-      are running:
-      <p><code>/etc/init.d/gefusion stop</code></p>
-      <p>/etc/init.d/geserver stop</p>
-    </li>
-    <li>Navigate to the <code>/opt/install/GoogleEarthFusion_Uninstaller</code> or <code>/opt/install/GoogleEarthServer_Uninstaller/</code> directory.
-      <p>To uninstall using the Fusion DVD, insert the DVD and mount the drive.
-        The uninstallers listed below are available at the top level of the
-        DVD media.</p>
-    </li>
-    <li>To uninstall Fusion Pro:<br>
-      <code>./Uninstall_Google_Earth_Enterprise_Fusion_GUI.sh // GUI uninstaller<br>
-        ./Uninstall_Google_Earth_Enterprise_Fusion.sh // Command line uninstaller</code>
-        </li>
-    <li>To uninstall Server:<br>
-      <code>./Uninstall_Google_Earth_Enterprise_Server_GUI.sh // GUI uninstaller<br>
-        ./Uninstall_Google_Earth_Enterprise_Server.sh // Command line uninstaller</code>
-      <p>The uninstall script starts. Follow the on-screen instructions to remove
-        the Google Earth Enterprise Fusion or Server products.</p>
-    </li>
-  </ol>
+<p>GEE provides a command line uninstall script.  To uninstall Google Earth Enterprise Fusion:</p>
+<ol>
+<li>Open a terminal window on your Linux workstation.</li>
+<li>Log in as <code>root</code>.</li>
+<li>Stop Google Earth Enterprise Fusion, if you have a previous version installed and it is running:
+<p><code>/etc/init.d/gefusion stop</code></p>
+</li>
+<li>Navigate to the location at which you initially cloned the GEE git repo.</li>
+<li>Within the GEE git repo, navigate to <code>earthenterprise/earth_enterprise/src/installer</code>
```

Comment:
Same comment as above on requiring the repository to uninstall an already installed Fusion.